src/Float.mo

```diff
@@ -538,23 +530,27 @@ module {
   ///
   /// Special cases:
   /// ```
-  /// notEqualWithin(+0.0, -0.0, epsilon) => false for any `epsilon >= 0.0`
-  /// notEqualWithin(-0.0, +0.0, epsilon) => false for any `epsilon >= 0.0`
-  /// notEqualWithin(+inf, +inf, epsilon) => false for any `epsilon >= 0.0`
-  /// notEqualWithin(-inf, -inf, epsilon) => false for any `epsilon >= 0.0`
-  /// notEqualWithin(x, NaN, epsilon)     => true for any x and `epsilon >= 0.0`
-  /// notEqualWithin(NaN, y, epsilon)     => true for any y and `epsilon >= 0.0`
+  /// notEqual(+0.0, -0.0, epsilon) => false for any `epsilon >= 0.0`
+  /// notEqual(-0.0, +0.0, epsilon) => false for any `epsilon >= 0.0`
+  /// notEqual(+inf, +inf, epsilon) => false for any `epsilon >= 0.0`
+  /// notEqual(-inf, -inf, epsilon) => false for any `epsilon >= 0.0`
+  /// notEqual(x, NaN, epsilon)     => true for any x and `epsilon >= 0.0`
+  /// notEqual(NaN, y, epsilon)     => true for any y and `epsilon >= 0.0`
   /// ```
   ///
   /// Example:
   /// ```motoko
   /// import Float "mo:base/Float";
   ///
   /// let epsilon = 1e-6;
-  /// Float.notEqualWithin(-12.3, -1.23e1, epsilon) // => false
+  /// Float.notEqual(-12.3, -1.23e1, epsilon) // => false
   /// ```
-  public func notEqualWithin(x : Float, y : Float, epsilon : Float) : Bool {
-    not equalWithin(x, y, epsilon)
+  public func notEqual(x : Float, y : Float, epsilon : Float) : Bool {
```

Comment:
Do we even want notEqual, since its the literal negation of equals @luc-blaeser ?

Reply:
Seems reasonable to remove this function. I'll merge and open another PR to start working on the merge conflicts in parallel.